src/Microsoft.Azure.WebJobs.Extensions.Dapr/Bindings/Converters/DaprStateConverter.cs

```diff
@@ -132,6 +135,15 @@ public DaprStateConverter(IDaprServiceClient daprClient)
         private async Task<string> GetStringContentAsync(DaprStateAttribute input, CancellationToken cancellationToken)
         {
             DaprStateRecord stateRecord = await this.GetStateRecordAsync(input, cancellationToken);
+
+            if (stateRecord.ContentStream.Length == 0)
```

Comment:
Shall we have similar checks for other places also? PubSub not found, Secret not found, etc. (Basically whatever Dapr errors out on). We can track it separately also.

Reply:
Thats a good idea, we can add for all of them.

I checked secret not found, and dapr does throw exception with details shown below, I believe that should be fine for secret. Rest of them we can handle if required.

`Exception while executing function: RetrieveSecretLocal. Microsoft.Azure.WebJobs.Host: Exception binding parameter 'secret'. Microsoft.Azure.WebJobs.Extensions.Dapr: "failed getting secret with key my-secret1 from secret store localsecretstore: secret my-secret1 not found".`